src/Database/PostgreSQL/Simple/Migration.hs

```diff
@@ -156,7 +156,7 @@ executeDirectoryMigration con opts dir =
 -- | Lists all files in the given 'FilePath' 'dir' in alphabetical order.
 scriptsInDirectory :: FilePath -> IO [String]
 scriptsInDirectory dir =
-  getDirectoryContents dir <&> (sort . filter (\x -> not $ "." `isPrefixOf` x))
+  sort <$> listDirectory dir
```

Comment:
Removal of . and .. is already done by [listDirectory](https://hackage.haskell.org/package/directory-1.3.8.1/docs/System-Directory.html#v:listDirectory)